### ISSUE TITLE
Syntax incompatibility with neo4j on querying edges by multiple labels

### ISSUE THREAD

**Issue by sebbsn** [SteelBridgeLabs/neo4j-gremlin-bolt#98]
neo4j-gremlin-bolt: v0.4.3
neo4j: v4.1.1

When creating one of the following queries I get an exception from Neo4j noting that separating alternative relationships by colon is not supported anymore (afaik since 4.x):

```
graph.traversal().V().hasLabel("vLabel").in(List.of("label1","label2").toArray(new String[0]))
graph.traversal().V().hasLabel("vLabel").out(List.of("label1","label2").toArray(new String[0]))
```

the corresponding error message:

```
i.r.p.w.a.GlobalDefaultExceptionHandler  : Sending ErrorResponse (status: 500 INTERNAL_SERVER_ERROR, ClientException: The semantics of using colon in the separation of alternative relationship types in conjunction with the use of variable binding, inlined property predicates, or variable length is no longer supported.
Please separate the relationships types using `:A|B|C` instead (line 1, column 47 (offset: 46))
"MATCH (n:`vLabel`)<-[r:`label1`|:`label2`]-(m) WHERE ID(n) = $id RETURN m"
```

Is this known behaviour? I currently workaround by using one label at a time, but I'd very much like to do it in a single query.

**Comment by rjbaucells** [comment]
The issue should be fixed, please try version `0.4.4` and close issue. Thanks for reporting it.

**Comment by sebbsn** [comment]
Works perfectly now without issues. Thanks a lot for the fast response!!!
